gprofiler/utils/__init__.py

```diff
@@ -207,7 +221,7 @@ def run_process(
     via_staticx: bool = False,
     check: bool = True,
     timeout: int = None,
-    kill_signal: signal.Signals = signal.SIGKILL,
+    kill_signal: signal.Signals = signal.SIGTERM,
```

Comment:
Revert?

Reply:
No. Windows does not support SIGKILL. I have added is_windows() check to select between both options.